### PR TITLE
.NET 8 Superpowers - updating agenda

### PR DESCRIPTION
**Pull request by sethdaily**
As per email: .NET 8 Superpowers - Agenda Update

### CODE REVIEW

File: content/events/net-superpowers-tour.mdx [hunk 1]
```diff
@@ -137,10 +141,15 @@ _body:
...
             content={<>
-              Cloudy with a Chance of Mobile
+              Cloud Native Development
             </>}
             afterBody={<>
-              * Write once and run anywhere with .NET
+              Building a production-ready distributed application that is resilient, observable, and hosted in the cloud
+              
+              * DAPR
+              * .NET
```
Comment (william-liebenberg) [inline content/events/net-superpowers-tour.mdx]: .NET Aspire is one thing -- not two :)

**Review by william-liebenberg — CHANGES_REQUESTED** [review]
Awesome - just need 1 fix please.

File: content/events/net-superpowers-tour.mdx (hunk 1 above)
Comment (sethdaily) [reply content/events/net-superpowers-tour.mdx]: > .NET Aspire is one thing -- not two :)
Done - merged into one

**Review by william-liebenberg — APPROVED** [review]
LGTM

**Pull request merged**